src/features/InstantMessagingProtocol/signalingServer/signalingServer.ts

```
@@ -167,7 +187,7 @@ export class SignalingServer {
                     log.debug("[IM] Received a register message")
                     // Validate the message schema
                     log.debug(data)
-                    var registerMessage: ImRegisterMessage =
+                    let registerMessage: ImRegisterMessage =
                         data as ImRegisterMessage
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)